modules/installation-special-config-encrypt-disk-tang.adoc

```diff
@@ -14,18 +14,26 @@ settings and run `openshift-install` to install a cluster and `oc` to work with
 for instructions. See link:https://youtu.be/2uLKvB8Z5D0[Securing Automated Decryption New Cryptography and Techniques]
 for a presentation on Tang.
 
-. Add kernel arguments to configure networking when you do the RHCOS installations for your cluster.
-If you miss this step, the second boot will fail.
-For example, to configure DHCP networking, identify `ip=dhcp`
-or set static networking when you add parameters to the kernel command line.
+. To configure networking when you do the {op-system-first} installations for your cluster, do one of the following:
 
+* Use the `ip=dhcp` kernel argument for DHCP networking.
+
+* Add the necessary `ip=` kernel arguments for static networking.
+
+[IMPORTANT]
+====
+Skipping this step causes the second boot to fail.
```

Comment:
Is this admonition in reference to the `rd.neednet=1` kernel argument? If so, I think it should be moved to the relevant step below (currently step 8).

Reply:
Similar to my previous comment, this statement strayed from original intent. I've removed it in the later `rd.neednet=1` example, and leaving it here where it was originally found. (Though before it came in the body text, and not in an admonition block.)

Reply:
Ok thanks Bob for clarifying that.